lib/matplotlib/tests/test_ticker.py

```diff
@@ -169,7 +169,7 @@ def test_SymmetricalLogLocator_set_params():
     nose.tools.assert_equal(sym.numticks, 8)
 
 
-@cleanup
+@cleanup(style='classic')
```

Comment:
Fortunately,  it doesn't seem to matter much.  style='classic' is needed two places in test_ticker, and a few more in the specific backend testers.   The tests are working as they are.  I don't think it is worth spending any time on right now.